opentelemetry/src/lib.rs

```diff
@@ -109,14 +158,15 @@
 //! * `trace`: Includes the trace API.
 //! * `metrics`: Includes the metrics API.
 //! * `logs`: Includes the logs bridge API.
+//! * `internal-logs`: Includes internal logging for the OpenTelemetry library via `tracing`.
 //!
-//! The default feature flags are ["trace", "metrics", "logs"]
+//! The default feature flags are ["trace", "metrics", "logs", "internal-logs"].
 //!
 //! The following feature flags provides additional configuration for `logs`:
 //! * `spec_unstable_logs_enabled`: Allow users to control the log level
 //!
 //! The following feature flags enable APIs defined in OpenTelemetry specification that is in experimental phase:
-//! * `otel_unstable`: Includes unstable APIs.
+//! * `otel_unstable`: Includes unstable APIs. There are no features behind this flag at the moment.
```

Comment:
nit - not related to this PR, but line 171 should be under separate section, and not subsection of `logs`.